examples/address/address_topic_auto_remove.yaml

```
@@ -6,5 +6,5 @@ spec:
   addressName: emptyAddress
   queueName: ""
   routingType: multicast
-  removeFromBrokerOnDelete: true
+  removeFromBrokerOnDelete: true  # optional
```

```suggestion
  removeFromBrokerOnDelete: true
```
The `# optional` comment it is not clear what it means, I would remove it
What about renaming the filename to `address_topic.yaml`?

---

Removed.